sentry-logback/src/main/java/io/sentry/logback/SentryAppender.java

```diff
@@ -33,6 +34,7 @@
 /** Appender for logback in charge of sending the logged events to a Sentry server. */
 @Open
 public class SentryAppender extends UnsynchronizedAppenderBase<ILoggingEvent> {
+  // WARNING: Do not use these options in here, they are only to be used for startup
```

Comment:
I think we should remove this options altogether and set the values directly in the configuration callback when initing the SDK, unless there's a reason to not do so.

Reply:
See comment above regarding #2053